frontend/src/components/SearchAddress.js

```diff
@@ -0,0 +1,26 @@
+import Address from './Address'
+
+const SearchAddress = () => {
+  return (
+  <div className="container-md" style={{marginTop: "30px"}}>
+    <h2 className="title-search-form">Where?</h2>
+    <form className="search-form" style={{marginTop: "30px"}}>
+      <div className="row mb-3">
+        <label htmlFor="From:" className="col-sm-2 col-form-label">From:</label>
```

Comment:
which element contains the `From:` id? I would suggest a different name that contains no special characters like ":" and uses camelCase. You could think about just putting the input field inside the label though, which has the same effect as providing a htmlFor prop